classes/Kohana/Arr.php

```diff
@@ -8,7 +16,7 @@
  * @copyright  (c) 2007-2014 Kohana Team
  * @license    http://kohanaframework.org/license
  */
-abstract class Kohana_Arr {
+class Arr {
```

Comment:
@rjd22 although I guess it feels neater without it, and in keeping with the other static-method-helper classes which I now see were already not defined as abstract, so :+1: from me. Sorry.